src/System.Management.Automation/resources/Modules.resx

```diff
@@ -231,6 +231,21 @@
   <data name="RequiredModuleNotLoadedWrongMinimumVersionAndMaximumVersion" xml:space="preserve">
     <value>The required module '{1}' with MinimumVersion '{2}' and MaximumVersion '{3}' is not loaded. Load the module or remove the module from 'RequiredModules' in the file '{0}'.</value>
   </data>
+  <data name="RequiredModuleNotFoundModuleVersion" xml:space="preserve">
+    <value>The module '{0}' cannot be found with ModuleVersion '{1}'</value>
```

Comment:
Please add final dots in new resource strings.

Reply:
@iSazonov Added here a17b8c5b9870f3b389ded39a1f8efba78a58041c